x/leverage/keeper/collateral.go

```diff
@@ -78,9 +78,59 @@ func (k Keeper) GetBorrowerCollateral(ctx sdk.Context, borrowerAddr sdk.AccAddre
 	return totalCollateral
 }
 
-// GetEligibleLiquidationTargets returns a list of borrower addresses eligible for liquidation
+// GetEligibleLiquidationTargets returns a list of borrower addresses eligible for liquidation.
 func (k Keeper) GetEligibleLiquidationTargets(ctx sdk.Context) ([]sdk.AccAddress, error) {
-	// TODO: Complete this to support liquidation tool
 	// ref: https://github.com/umee-network/umee/issues/229
-	return nil, nil
+	store := ctx.KVStore(k.storeKey)
+	borrowPrefix := types.CreateLoanKeyNoAddress()
+
+	iter := sdk.KVStorePrefixIterator(store, borrowPrefix)
+	defer iter.Close()
+
+	liquidationTargets := []sdk.AccAddress{}
+	checkedAddrs := map[string]bool{}
```

Comment:
nit: If you don't care about the actual value and you're using the map as more of a set data structure, it's better to use `struct{}` as the value -- this provides the benefit of zero allocations over a bool.

```suggestion
	checkedAddrs := map[string]struct{}
```